README.md

```diff
@@ -1,3 +1,36 @@
-This is a nascent project for implementing a rich text layout engine backed
-by the [swash](https://github.com/dfrg/swash) crate and implementing the 
-[piet](https://github.com/linebender/piet) text API.
+<div align="center">
+
+# Parley
+
+**Rich text layout**
+
+[![Latest published parley version.](https://img.shields.io/crates/v/parley.svg)](https://crates.io/crates/parley)
+[![Documentation build status.](https://img.shields.io/docsrs/parley.svg)](https://docs.rs/parley)
+[![Dependency staleness status.](https://deps.rs/repo/github/linebender/parley/status.svg)](https://deps.rs/repo/github/linebender/parley)
+[![Linebender Zulip chat.](https://img.shields.io/badge/Linebender-%23gpu-blue?logo=Zulip)](https://xi.zulipchat.com/#narrow/stream/205635-text)
+[![Apache 2.0 or MIT license.](https://img.shields.io/badge/license-Apache--2.0_OR_MIT-blue.svg)](#license)
+
+</div>
+
+Parley provides the [Piet](https://github.com/linebender/piet) text API for implementing rich text layout.
+It is backed by [Swash](https://github.com/dfrg/swash).
+
+## Community
+
+Discussion of Parley development happens in the [Linebender Zulip](https://xi.zulipchat.com/), specifically the [#text stream](https://xi.zulipchat.com/#narrow/stream/205635-text).
+All public content can be read without logging in.
+
+Contributions are welcome by pull request. The [Rust code of conduct] applies.
+
+Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in the work by you, as defined in the Apache 2.0 license, shall be licensed as noted in the [License](#license) section, without any additional terms or conditions.
```

Comment:
I don't much like the move away from following the [Rust API guidelines](https://rust-lang.github.io/api-guidelines/necessities.html#crate-and-its-dependencies-have-a-permissive-license-c-permissive) text exactly here, but the change was approved for Vello, so it's *fine*

Reply:
Yeah I don't really have an opinion here, I just copied what Vello had as it was the newest, plus made the "License" reference into an actual link.